sv_interval.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, chrom=None, start=0, end=0, name=None, sv_type=None, length=0
     self.is_precise = False
     self.is_validated = False
     self.validating_interval = None
-    self.info_dict = {}
+    self.info_dict = {} # Marghoob, did you have any specific plans for this?
```

Comment:
@marghoob did you have have specific plans for this as opposed to "info"? Was it just supposed to mirror info?